docs/spec/README.md

```diff
@@ -0,0 +1,18 @@
+# Knative Eventing API spec
+
+This directory contains the specification of the Knative Eventing API, which is
+implemented in [`channels.knative.dev`](/pkg/apis/channels/v1alpha1) and
+[`feeds.knative.dev`](/pkg/apis/feeds/v1alpha1) and verified via [the e2e
+test](/test/e2e).
+
+**Updates to this spec should include a corresponding change to the
+API implementation for [channels](/pkg/apis/channels/v1alpha1) or
+in [feeds](/pkg/apis/feeds/v1alpha1) and [the e2e test](/test/e2e).**
```

Comment:
This may be the right place, but it seems like it would be useful to include a section on either "Known issues" with the API and/or patterns that we've agreed upon. In particular, the use of containers/images for extension of Buses and Sources is probably worth highlighting somewhere.

The Parameters/ParametersFrom pattern for passing args to the images is probably also worth documenting.

Reply:
I think that is a good TODO but I don't have the context to fully author those. I added a todo for you :D